### ISSUE TITLE
k8s check status wait tasks fail on OCP3.7

### ISSUE THREAD

**Issue by fbladilo** [fusor/ocp-mig-test-data#35]
These tasks fail consistently on OCP3.7 and it is not possible to validate under that release in the current state. Perhaps @fabianvf  can help in diagnosing why they fail or a possible workaround fix. Another alternative could be utilizing a different method for checking status with a conditional only for OCP3.7

The task in question is the following [https://github.com/fusor/ocp-mig-test-data/blob/master/roles/deployment/nginx_deployment/tasks/check.yml](url)

Sample failure : 
```
TASK [basic_app_examples/nginx_backup_restore : Create Nginx application] ******

changed: [localhost]

TASK [basic_app_examples/nginx_backup_restore : Wait 2 minutes for nginx deployment] ***

FAILED - RETRYING: Wait 2 minutes for nginx deployment (10 retries left).

FAILED - RETRYING: Wait 2 minutes for nginx deployment (9 retries left).

FAILED - RETRYING: Wait 2 minutes for nginx deployment (8 retries left).

FAILED - RETRYING: Wait 2 minutes for nginx deployment (7 retries left).

FAILED - RETRYING: Wait 2 minutes for nginx deployment (6 retries left).

FAILED - RETRYING: Wait 2 minutes for nginx deployment (5 retries left).

FAILED - RETRYING: Wait 2 minutes for nginx deployment (4 retries left).

FAILED - RETRYING: Wait 2 minutes for nginx deployment (3 retries left).

FAILED - RETRYING: Wait 2 minutes for nginx deployment (2 retries left).

FAILED - RETRYING: Wait 2 minutes for nginx deployment (1 retries left).

fatal: [localhost]: FAILED! => {"attempts": 10, "changed": false, "resources": []}

PLAY RECAP *********************************************************************
localhost                  : ok=2    changed=1    unreachable=0    failed=1   
```
I have not investigated much further but opening issue to keep track. It might be that we need to just alter the current task. @alexander-demichev

**Comment by alexander-demicev** [comment]
@fabianvf Hi, can we have your opinion on this?

**Comment by fabianvf** [comment]
It's hard to say, it might just need an increased timeout in 3.7 for some reason. Can we see what the deployment looks like?

edit: I mean the state of it when we're failing, not the definition

**Comment by fbladilo** [comment]
Closing addressed by #40
